Library/Homebrew/test/support/helper/test_case.rb

```diff
@@ -12,6 +12,20 @@ class TestCase < ::Minitest::Test
     TEST_SHA1   = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef".freeze
     TEST_SHA256 = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef".freeze
 
+    def before_setup
```

Comment:
Heh, I hate it when I spend ages trying and failing to work out a problem, then come back to it hours later and immediately get it. But also I love it. 😁